### PR TITLE
Properties and parameters UI sugar

### PR DESCRIPTION
**Pull request by didimitrie**
Due to changes in the new connectors, we've had to spice up and re-add some extra love for properties and params. This is a hopefully small PR!

### CODE REVIEW

File: packages/frontend/src/main/components/viewer/ViewerFilters.vue [hunk 1]
```diff
@@ -201,8 +201,10 @@ export default {
         filter.data = rawFilter
         const key = rawFilter.key
         // Handle revit params (a wee bit of FML moment)
-        if (key.startsWith('parameters.')) {
+        if (key.startsWith('parameters.') || key.startsWith('properties.')) {
           if (key.endsWith('.value')) {
+            console.log('pushing', key)
+            console.log('pushing', key)
```
Comment (andrewwallacespeckle) [inline packages/frontend/src/main/components/viewer/ViewerFilters.vue:207]: Did you mean to log this twice?

Comment (didimitrie) [reply packages/frontend/src/main/components/viewer/ViewerFilters.vue:207]: i probably meant to not log this at all! i'll fix!

**Review by andrewwallacespeckle — APPROVED** [review]
All looks good. I'm having a problem with my local dev environment, so couldn't test it (will do it once I fix it), but I'm sure you have tested anyway, so feel free to go ahead and merge if you want it in

**Comment by didimitrie** [comment]
> All looks good. I'm having a problem with my local dev environment, so couldn't test it (will do it once I fix it), but I'm sure you have tested anyway, so feel free to go ahead and merge if you want it in

yep - have sanity checked against current revit models, past and present! i'll merge away then.

**Pull request merged**